src/Behavioral.Automation/FluentAssertions/Abstractions/IAssertionAccessor.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Behavioral.Automation.Model;
+
+namespace Behavioral.Automation.FluentAssertions.Abstractions
+{
+    public interface IAssertionAccessor
```

Comment:
i dont like this interface. am i right that method 'Validate' intended to perform validation and all get-accessros are exposing validation result? can we have 'Validate' method which returns DTO with the whole validation result data?